<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QMainWindow" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="plotLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClear_2">
    <property name="title">
     <string>clear plot</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReset_plot_zoom">
    <property name="title">
     <string>reset plot zoom</string>
    </property>
   </widget>
   <addaction name="menuClear_2"/>
   <addaction name="menuReset_plot_zoom"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action0_0">
   <property name="text">
    <string>0 0</string>
   </property>
  </action>
  <action name="action0_1">
   <property name="text">
    <string>0 0</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
